packages/wrangler/src/bundle.ts

```diff
@@ -57,6 +57,9 @@ export async function bundleWorker(
     sourcemap: true,
     metafile: true,
     conditions: ["worker", "browser"],
+    define: {
+      "process.env.NODE_ENV": `"${process.env.NODE_ENV}"`,
```

Comment:
I think we would need to escape `"` found in `process.env.NODE_ENV`, just in case.